packages/dd-trace/src/proxy.js

```diff
@@ -50,7 +54,23 @@ class Tracer extends NoopProxy {
         // dirty require for now so zero appsec code is executed unless explicitly enabled
         if (config.appsec.enabled) {
           require('./appsec').enable(config)
+        } else if (config.appsec.enabled === undefined) { // only activate remote config when conf is not set locally
+          rc.updateCapabilities(RemoteConfigCapabilities.ASM_ACTIVATION, true)
+
+          rc.on('ASM_FEATURES', (action, conf) => {
+            if (conf && conf.asm && typeof conf.asm.enabled === 'boolean') {
+              if (action === 'apply' || action === 'modify') action = conf.asm.enabled // take control
```

Comment:
Let's always use curly braces except for short-circuits to make the code clearer.